pkg/openservicebroker/server/apiserver.go

```diff
@@ -0,0 +1,73 @@
+package server
+
+import (
+	kclientsetinternal "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	templateapi "github.com/openshift/origin/pkg/template/apis/template"
+	templateinformer "github.com/openshift/origin/pkg/template/generated/informers/internalversion"
+	templateservicebroker "github.com/openshift/origin/pkg/template/servicebroker"
+	genericapiserver "k8s.io/apiserver/pkg/server"
+)
+
+type TemplateServiceBrokerConfig struct {
+	GenericConfig *genericapiserver.Config
+
+	// these are only needed for the template service broker, which should move out
+	KubeClientInternal kclientsetinternal.Interface
+	TemplateInformers  templateinformer.SharedInformerFactory
+	TemplateNamespaces []string
+}
+
+// TemplateServiceBrokerServer serves non-API endpoints for openshift.
```

Comment:
> copy/paste of comment? not really specific

It's how linting improves our code.  Something I learned about in my intro to java class.  We've been doing it wrong all these years. :)